### ISSUE TITLE
2.23.2 crashing on startup!

### ISSUE THREAD

**Issue by simonhermann** [WhisperSystems/TextSecure#3783]
Used to worked great, but after last update crashes on startup on my Moto X 2013 with Android 5.1.
Tried to uninstall, but now it crashes when I try to register..
please fix that as soon as possible!
Thanks,
Simon

**Comment by moxie0** [comment]
Hey @simonhermann sorry to hear that.  Are you capable of getting logcat output for this crash?  You'd have to use something like 'adb' while the device is connected to your desktop.

**Comment by simonhermann** [comment]
Hi, thanks for your quick reply!
After a few reboot and re-installations I got it somehow working again.. 
I could submit the in-app crash log, or shall I post it here?

**Comment by moxie0** [comment]
Thanks @simonhermann but there's nothing in that log, you'd need to capture the debug log immediately after the crash.  Out of curiosity, can you hit the "+" floating action bar to start a new conversation in the conversation list, then "pull to refresh" on your list of contacts?  Does that cause a crash, or does it work fine?

**Comment by simonhermann** [comment]
I just tried refreshing the contatcs list, this and everything else seems to work perfectly fine now. 
If I get a crash again, I'll try to get the log via this 'adb' before uninstalling..
thank you for your help anyway @moxie0 ! 

I just realised that I lost all of my chat logs during the process, It's my own fault since I did not use the export function. I suppose it is not possible to pop up a warning before uninstalling and point out the import/export?
